.scalafmt.conf

```diff
@@ -3,7 +3,7 @@ runner.dialect = scala212
 
 maxColumn = 120
 
-align.preset = some
+align.preset = none
```

Comment:
I quite liked `some` here, but I'd like to hear your opinion on this one (i.e. why `none` ? )

Reply:
It definitely may look nice in some instances, but it creates unnecessary changes in diff. It also may create lines longer than the limit, where it would not happen without it.

Those are not huge issues, so I am ok using 'some' if you think it's worth it.